Lundgren-2020-275

Kristina-Pianykh opened the pull request:
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [x] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
1. A new wordlist (see issue #808)
2. A new concept set CATFISH
3. 'catfish' glosses in other lists are now mapped to CATFISH
4. 'mourn' in Hale is now mapped to GRIEVE

Concerning the mapping in Lundgren-2020-275:
Many glosses are vague and not narrowed down to a specific concept (e.g., BACK (part of the body or side of an object?), BE FAT (=obese or in relation to meat?), DRIP (INTR.) (could be both 808 and 2253), etc.)

## Question 
The gloss NECK~THROAT suggests a potentially new concept NECK OR THROAT (see also a similar gloss in Payne-1991-202). Shall I add it now in this PR or rather open an issue to discuss it in more detail?

Kristina-Pianykh commented:
Any volunteers to manage the PR? Christoph @chrzyki, do you mind if I pin you here?

chrzyki commented:
Sure, happy to do it. @Schweikhard, @CarolinHu, @AnnikaTjuka, would you mind reviewing?

Schweikhard commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-62	62	62	COLLARED PECCARY	435	COLLARED PECCARY
+Lundgren-2020-275-63	63	63	COME (INTR.)	1446	COME
+Lundgren-2020-275-64	64	64	COMPANION	1325	FRIEND
+Lundgren-2020-275-65	65	65	COOK	1100	COOK (SOMETHING)
```
This could also be the person.

LinguList replied:
In fact, by checking the original data, you may be able to find that out, as the concrete words may give hints to part of speech.

Kristina-Pianykh replied:
This must be a verb (tracing the original lexical data of Kokama to the [GRAMMAR OF KOKAMA-KOKAMILLA by Vallejos (2010)](file:///C:/Users/krist/Desktop/VallejosYopn_Rosa_phd2010fa.pdf).

Schweikhard commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-61	61	61	COCKROACH	1575	COCKROACH
+Lundgren-2020-275-62	62	62	COLLARED PECCARY	435	COLLARED PECCARY
+Lundgren-2020-275-63	63	63	COME (INTR.)	1446	COME
+Lundgren-2020-275-64	64	64	COMPANION	1325	FRIEND
```
Not 100% sure whether to map it to that. We have two more concept lists with "companion" that are unmapped (and two with "friend, companion" mapped to friend).

LinguList replied:
Do the unmapped companions have a friend in the SAME list? If not, we could map them all to friend for now, as the rule would also say: if you don't have a list distinguishing it, you can map it (exceptions our new ratings lists of 3000 + items, as they will surely have both...).

Schweikhard replied:
They have. In one of the cases it even has the note that it's the same meaning, though.

Kristina-Pianykh replied:
Since the ambiguous concepts with ~ should be mapped by the first item in a pair as advised by Mattis, we now have two FRIEND instances (i.e. COMPANION and FRIEND ~ LOVER). In this case, I'm inclined to leave COMPANION mapped to FRIEND while FRIEND ~ LOVER - unmapped (also the current definition of FRIEND explicitly excludes lovers).

Schweikhard commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-67	67	67	CORN	506	MAIZE
+Lundgren-2020-275-68	68	68	COTTON	1850	COTTON
+Lundgren-2020-275-69	69	69	COVER (TR.)	761	COVER
+Lundgren-2020-275-70	70	70	CROSS	1367	CROSS
```
Or TRANSIT?

LinguList replied:
Yes, again, see my comment above, here, one needs to do detective work and go to the original text with the wordlist. Note also that when you do so, you should  have a look at what the author writes about the selection of concepts (maybe they had some extended swadesh lists in mind, etc.)

Kristina-Pianykh replied:
@Schweikhard you are right, this one is definitely a verb and it's used in the sense 'to cross a river/creek, etc.'

Schweikhard commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-70	70	70	CROSS	1367	CROSS
+Lundgren-2020-275-71	71	71	CRY (INTR.)	1839	CRY
+Lundgren-2020-275-72	72	72	CULTIVATE (TR.)	1541	CULTIVATE
+Lundgren-2020-275-73	73	73	CURE	499	CURE
```
verb or noun?

LinguList replied:
dito, what I mentioned above. Important catch, @Schweikhard

Kristina-Pianykh replied:
This is a difficult one. In Kokama this lexeme can act both as a noun and as a verb. In Omagua, however, this lexeme is translated as a noun (source: A Linguistic Analysis of Old Omagua Ecclesiastical Texts by Michael and O'Hagan (2016) (file:///C:/Users/krist/Desktop/A_Linguistic_Analysis_of_Old_Omagua_Eccl.pdf). There is more specific concept for CURE (NOUN) in the Concepticon and there are also a number of unmapped substantive 'cure' instances in other lists. I can add a new concept set then to refer to a medicine or a treatment as a whole (i.e. a hypernym to MEDICINE)

Schweikhard commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-88	88	88	DUST~SAND		
+Lundgren-2020-275-89	89	89	EAR	1247	EAR
+Lundgren-2020-275-90	90	90	EGG	744	EGG
+Lundgren-2020-275-91	91	91	ENCOUNTER		
```
MEET I suppose.

Schweikhard replied:
Unless it's also about encountering an object.

Kristina-Pianykh replied:
Interesting observation, the Kokama lexeme under this concept actually mean 'respond, answer' (A Linguistic Analysis of Old Omagua Ecclesiastical Texts by Michael and O'Hagan (2016), Pequeno vocabulário Português-Tupi (Barbosa, 1970)), while the Omagua lexeme can mean both 'respond' and 'encounter' (the latter in the sense of experiencing something as an event, not literally as in 'meet a person'). So if we are to stick to Lundgren, I'd leave this one unmapped.

Schweikhard commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-171	171	171	MOUTH	674	MOUTH
+Lundgren-2020-275-172	172	172	NAIL (BODY PART)	1896	FINGERNAIL OR TOENAIL
+Lundgren-2020-275-173	173	173	NAME		
+Lundgren-2020-275-174	174	174	NECK~THROAT		
```
Hm, NECK refers to the whole Hals, both NAPE OF NECK and THROAT, so having a concept NECK OR THROAT that refers to both the whole and a part seems a bit strange (if not unheard of).

LinguList replied:
If there's no neck in the data, link to neck.

LinguList commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-220	220	220	SMOKE FOOD		
+Lundgren-2020-275-221	221	221	SMOOTH	1234	SMOOTH
+Lundgren-2020-275-222	222	222	SNAKE	730	SNAKE
+Lundgren-2020-275-223	223	223	SON (M. EGO)		
```
This is an important concept that we should add. It is SON (OF MALE), so it is a kinship term, similar to the ones we have in "SISTER (OF MALE PERSON)". They use "ego" to denote "person". So I suggest to add DAUGHTER and SON (OF MALE EGO) to concepticon (and on the long run, we should switch our nomenclature on kinship terms also. 

This requires to add a relation between SON which is broader than SON (OF MALE EGO) etc.

Kristina-Pianykh replied:
Agreed. What about other kinship terms in relation to SISTER-IN-LAW (F. EGO) and GRANDCHILD (F. EGO)? Shall I add the corresponding concepts for them too?

LinguList commented on concepticondata/references/references.bib:
```diff
@@ -3121,4 +3121,18 @@ @Book{Hartmann2013
-}    
+}
+
+@misc{Lundgren2020,
```
instead of misc, we use "thesis" or "mastersthesis" in bibtex.

See the bibtex I provided [here](https://github.com/lexibank/lexibank/issues/208).

LinguList replied:
and sorry for not mentioning this, as this would have saved some work.

LinguList commented:
Good point, @Kristina-Pianykh, I support your decision.

CarolinHu commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-22	22	22	BATHE (INTR.)	138	BATHE
+Lundgren-2020-275-23	23	23	BE BORN (INTR.)	1090	BE BORN
+Lundgren-2020-275-24	24	24	BE DRY	1398	DRY
+Lundgren-2020-275-25	25	25	BE FAT		
```
1279 FAT (OBESE)?

Kristina-Pianykh replied:
I left this one unmapped, because it's unclear if the concept refers to the description of a human body (and in Kokama it is) or to meat/oil. I can't find more information on the meaning of the corresponding lexemes in the other languages.

CarolinHu commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-84	84	84	DOOR	1567	DOOR
+Lundgren-2020-275-85	85	85	DRIP (INTR.)		
+Lundgren-2020-275-86	86	86	DRUNK	1463	DRUNK
+Lundgren-2020-275-87	87	87	DRY (V.)		
```
2015 MAKE DRY?

Kristina-Pianykh replied:
Here, too, I can't find out more about this concept because this gloss can refer both to the transitive and intransitive uses (MAKE DRY and DRY UP).

CarolinHu commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-187	187	187	PATIO		
+Lundgren-2020-275-188	188	188	PEANUT	2364	PEANUT
+Lundgren-2020-275-189	189	189	PERSON	683	PERSON
+Lundgren-2020-275-190	190	190	PET		
```
This (or something like domesticated animal) could perhaps be an interesting concept to add as there is already the counterpart in Concepticon, WILD ANIMAL.

Kristina-Pianykh replied:
Good point, I'll add then PET to concepticon.tsv.

Kristina-Pianykh commented:
I've updated the bibtex record as well as some mappings in Lundgren according to the discussions above. Regarding the pairwise concepts with ~, I mapped them by the first item where possible. I've also introduced new concept sets: PET, SON (OF MALE EGO) and DAUGHTER (OF MALE EGO); and updated the concept relations.

AnnikaTjuka commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-232	232	232	STEM	2628	PLANT STEM
+Lundgren-2020-275-233	233	233	STONE	857	STONE
+Lundgren-2020-275-234	234	234	STRAIN (TR.)	940	STRAIN
+Lundgren-2020-275-235	235	235	SOUND (V.)		
```
2879 SOUND (OF INSTRUMENT OR VOICE) ?

Kristina-Pianykh commented on concepticondata/concepticon.tsv:
```diff
@@ -3760,3 +3760,6 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3776	TWO THOUSAND AND TWO	Quantity	The natural number two thousand and two (2002).	Number	
+3777	CATFISH	Animals	Any of ray-finned fish with long stiff barbels, resembling a cat's whiskers. There are diverse types of catfish, ranging in size and appearance and inhabiting mostly freshwater environments.	Person/Thing	
+3778	DAUGHTER (OF MALE EGO)	Kinship	A daughter, as referred to by her father.	Person/Thing	
+3779	SON (OF MALE EGO)	Kinship	A son, as referred to by his father.	Person/Thing	
```
So now I'm officially confused. The original motivation behind the naming was the following: the 'ego' left by the author in parenthesis means that this concept relates to one specific instance of referring to a person as your daughter/son if you are yourself a man (i.e. father). Following this argumentation, another UNRELATED MALE person can't call the person referred to as daughter or son with these lexemes. This is what I thought refers to by 'ego'. Or I'm just overthinking :)

AnnikaTjuka replied:
I thought it was similar to BROTHER (F. EGO) 560 BROTHER (OF WOMAN) but you could also be right.

Kristina-Pianykh replied:
Alright, once I've looked at the definition of BROTHER (OF WOMAN), I've understood what you mean. The reason why the naming involving (OF WOMAN) can be potentially ambiguous (at least for me, maybe it's not the case for others), is that it can mean:

1. referring to the person in question if you are a woman. In this case, without looking at the definition of BROTHER (OF WOMAN), I would assume this concept can be used by potentially any woman when referring to somebody's brother.

2. referring to the person in question from the ego (deictic) perspective. In this case BROTHER (OF WOMAN) would mean that this concept can be used exclusively by the sister of the person in question.

I hope I didn't confuse things even more :)

AnnikaTjuka replied:
Kinship terms are confusing by nature ;) Most systems are far more intricate than English and therefore, we lack a good term to describe the exact relationship. But I'd propose you follow @LinguList suggestion to use the "ego" style for now.

CarolinHu commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-116	116	116	GO DOWNRIVER (INTR.)		
+Lundgren-2020-275-117	117	117	GOURD	411	GOURD
+Lundgren-2020-275-118	118	118	GRAB	855	SEIZE
+Lundgren-2020-275-119	119	119	GRANDCHILD (F. EGO)		
```
If I understand it correctly, these are different concepts. GRANDCHILD (F. EGO) means the female person is at the level of the grandparents while 2703 implies a female person at the level of the parents of the child in question (while ego is a grandparent of unknown gender).

LinguList commented:
Yes, that would be very cool, as there is a kinship project in the
making and they complained we lacked the concepts here ;)

Kristina-Pianykh commented on concepticondata/conceptlists/Lundgren-2020-275.tsv:
```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-100	100	100	FISH (TR.)	2638	FISHING
+Lundgren-2020-275-101	101	101	FISH POISON	348	FISH POISON
+Lundgren-2020-275-102	102	102	FISHING NET	388	FISHNET
+Lundgren-2020-275-103	103	103	FLAME (V.)	1428	BURNING
```
In dictionaries and grammars of Kokama, the corresponding lexeme means something like 'be bright' but rather broadly. I didn't find more about the lexical items in the other target languages. I agree this is an ambiguous concept now that I think about it. I'll leave it unmapped.

LinguList commented:
@SimonGreenhill, given your involvement in kinbank and parabank, could
you clarify what the usage of "ego" is? Is it  just another word for
"person"? Or do you know whom we could ask?

LinguList commented:
No, I think that this is what this means. I guess, at some point we will
need to overhaul all kinship terminology in concepticon, but for now, it
may be best to just follow the "ego" style already. But I leave that to
the others to decide.

SimonGreenhill commented:
"ego" in this context is the referent, so "GRANDCHILD (F. EGO)" means "the term used by a female person to refer to their grandchild", i.e. what your Oma calls you, but not what Opa calls you.

SimonGreenhill commented:
BTW: kinbank is coming soon, see concepts here: https://github.com/SimonGreenhill/kinbank/blob/master/kinbank/cldf/parameters.csv

Kristina-Pianykh commented:
@SimonGreenhill this is absolutely mind-blowing as it only proves once again how poor English is when it comes to kinship terms (which is not such a bad thing now that I think about it!).

Kristina-Pianykh commented:
I updated the mappings and introduced the new concept sets according to the comments and discussions above. Let me know if there is anything else I could improve, I'd be happy to do so.

SimonGreenhill commented:
English is very impoverished in many ways :)

chrzyki commented:
Nice, thanks for all the contributions and for the neat list & concept refinements @Kristina-Pianykh. This can be merged as far as I'm concerned and will trigger a needed rebuild of `halenepal` (https://github.com/lexibank/halenepal/issues/15).
